pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -155,9 +155,11 @@ func (m *Multicluster) close() (err error) {
 	// Remove all of the clusters.
 	g, _ := errgroup.WithContext(context.Background())
 	for _, clusterID := range clusterIDs {
-		g.Go(func() error {
-			return m.DeleteMemberCluster(clusterID)
-		})
+		func(cid string) {
```

Comment:
prefer `clusterID:=clusterID` to avoid needing to nest it more

Reply:
+1. https://github.com/istio/istio/pull/32225